artemis/src/main/java/tech/pegasys/artemis/cli/deposit/CommonParams.java

```diff
@@ -85,6 +103,41 @@ public UnsignedLong getAmount() {
     return amount;
   }
 
+  Credentials getEth1Credentials() {
+    if (eth1PrivateKeyOptions.eth1PrivateKey != null) {
+      return Credentials.create(eth1PrivateKeyOptions.eth1PrivateKey);
+    }
+
+    final String keystorePassword =
+        KeystorePasswordOptions.readFromFile(
+            spec.commandLine(),
+            eth1PrivateKeyOptions.keystoreOptions.eth1PrivateKeystorePasswordFile);
+    final File eth1PrivateKeystoreFile =
+        eth1PrivateKeyOptions.keystoreOptions.eth1PrivateKeystoreFile;
+    try {
+      return WalletUtils.loadCredentials(keystorePassword, eth1PrivateKeystoreFile);
+    } catch (final FileNotFoundException e) {
+      throw new CommandLine.ParameterException(
+          spec.commandLine(), "Error: File not found: " + eth1PrivateKeystoreFile, e);
+    } catch (final IOException e) {
+      throw new CommandLine.ParameterException(
+          spec.commandLine(),
+          "Error: Unexpected IO Error while reading v3 keystore ["
```

Comment:
nit: Instead of saying "reading v3 keystore" I think just calling it an Eth1 keystore would be clearer so something like "reading Eth1 keystore"

Reply:
👍